orchestrationSpecs/packages/schemas/src/zodMeta.ts

```diff
@@ -0,0 +1,180 @@
+/**
+ * Zod GlobalMeta Type Extension
+ * 
+ * This module extends Zod's GlobalMeta interface to define UI metadata
+ * that flows through to OpenAPI/JSON Schema via @asteasolutions/zod-to-openapi.
+ * 
+ * Usage:
+ * ```typescript
+ * import 'zod';
+ * import './zodMeta'; // Side-effect import to extend GlobalMeta
+ * 
+ * const schema = z.string().meta({
+ *   title: "Field Title",
+ *   placeholder: "Enter value...",
+ *   group: "my-group",
+ *   order: 1
+ * });
+ * ```
+ */
+
+import 'zod';
+
+declare module 'zod' {
+    interface GlobalMeta {
+        // ============================================
+        // UI Presentation
+        // ============================================
```

Comment:
Since this is on the schema itself, I'm going to evaluate the metadata INDEPENDENTLY from the web-ui concerns.  Some of this is useful in many contexts and some is valuable only for a webpage.  My lens will be - could this information benefit a user directly.  e.g. do the semantics alone provide value to a user?